internal/lang/functions_test.go

```diff
@@ -1311,9 +1327,14 @@ func TestFunctions(t *testing.T) {
 
 			for _, test := range funcTests {
 				t.Run(test.src, func(t *testing.T) {
-					data := &dataForTests{} // no variables available; we only need literals here
+					data := &dataForTests{
```

Comment:
I guess in some later PR we might want to factor this "fake evaluation data for tests" idea out into a single shared location, since I notice we now have at least two variations of it! (One in `package lang`, and another in `package terraform`.)

That's for another day, though. Not important for this PR.